homeassistant/components/dsmr/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "dsmr",
-  "name": "DSMR Slimme Meter",
+  "name": "DSMR Smart Meter (Serial)",
```

Comment:
I'm not sure what "(Serial)" adds here; IMHO it is a technical term that demotes the communication method, which is odd to put into an integration name.

For example, we don't append the AdGuard integration with "(HTTP REST)" either.

Reply:
Wanted to differentiate it somewhat from DSMR Reader integration, which could renamed as DSMR Reader (MQTT). But I have no strong opinions on that